tools/test/scripts/onos-config.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+ONOS_CONFIG_ROOT=$HOME/go/src/github.com/onosproject/onos-config
+
+go run github.com/onosproject/onos-config/cmd/onos-config-manager \
+    -configStore=$ONOS_CONFIG_ROOT/configs/configStore-sample.json \
+    -changeStore=$ONOS_CONFIG_ROOT/configs/changeStore-sample.json \
+    -deviceStore=$ONOS_CONFIG_ROOT/configs/deviceStore-sample.json \
+    -networkStore=$ONOS_CONFIG_ROOT/configs/networkStore-sample.json \
+    -caPath $ONOS_CONFIG_ROOT/tools/test/devicesim/certs/onfca.crt \
```

Comment:
see updated run.md - no need for caPath, keyPath, certPath now

Reply:
Sure. I will update it accordingly.